### DISCUSSION TITLE
What insect is believed to have up to 12,000 eyes?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#515] [Galaxy Trivia] [answered]
```
   👀
 👀👀
👀👀👀
 👀👀
  👀
```

**Comment by NobleOsinachi** [comment]
Fly

**Comment by peterbaccaro** [comment]
The **Monarch Butterfly** is known to have 12,000 eyes.

**Comment by thuongtruong1009** [comment, marked as answer]
The insect that is believed to have up to 12,000 eyes is the dragonfly. Dragonflies have large compound eyes that cover most of their head, and each eye is made up of thousands of tiny lenses called ommatidia. These ommatidia work together to give dragonflies a wide field of vision and excellent depth perception, which is important for their aerial hunting. While the number of ommatidia can vary between species and individuals, some dragonflies are believed to have as many as 30,000 ommatidia in each eye, which adds up to a total of up to 60,000 lenses.
